addons/project/controllers/portal.py

```diff
@@ -77,7 +79,7 @@ def portal_my_project(self, project_id=None, **kw):
         return request.render("project.portal_my_project", vals)
 
     @http.route(['/my/tasks', '/my/tasks/page/<int:page>'], type='http', auth="user", website=True)
-    def portal_my_tasks(self, page=1, date_begin=None, date_end=None, sortby=None, filterby=None, search=None, search_in='content', **kw):
+    def portal_my_tasks(self, page=1, date_begin=None, date_end=None, sortby=None, filterby=None, search=None, search_in='content', groupby='project', **kw):
```

Comment:
As we target stable, modifying an API is not possible. You can do groupby = kw.get('groupby', 'project') in the code.